spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8BootstrapConfigurationTests.java

```diff
@@ -47,7 +47,8 @@ public void configAndSecretsBeansAreNotPresent() {
 	}
 
 	@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = Application.class,
```

Comment:
tbh, when this test started failing I was in a tiny shock. Before I added `"spring.cloud.kubernetes.client.namespace=default"`, it failed with `NamespaceResolutionFailedException`. And I  had the immediate reaction - "is this going to fail the same from now on for all users?" obviously this gave me a cold shower. 

After some time thinking about it, this is not the case. When running the application, users are going to be in a k8s environment, so the usual steps to resolve the namespace will happen.